dotnet/nuget/p528.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>P528</id>
+    <version>4.3.0</version>
+    <authors>The Institute for Telecommunication Sciences</authors>
+    <owners>The Institute for Telecommunication Sciences</owners>
+    <license type="file">LICENSE.md</license>
+    <projectUrl>https://github.com/NTIA/p528</projectUrl>
+    <iconUrl>https://www.its.bldrdoc.gov/media/66689/itslogosquare.png</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Recommendation ITU-R P.528: A propagation prediction method for aeronautical mobile and radionavigation services using the VHF, UHF and SHF bands.  This Recommendation contains a method for predicting basic transmission loss in the frequency range 125-15 500 MHz for aeronautical and satellite services. The software implements Annex 2 of Rec P.528, the Step-by-Step method to computing propagation loss for air-to-ground paths.</description>
```

Comment:
Yes.  The current P.528-4 is 125 - 15,500 MHz.  I wrote it with a space instead of a comma as that is the standard way of writing it in ITU-speak.  Its also how we wroteit in the README.md file for the repo.